src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function validationOfString(givenString: string, searchQuery: string) {
```

Comment:
better something like this
```suggestion
function doesStringMatchQuery(givenString: string, searchQuery: string) {
```